src/index.html

```diff
@@ -9,6 +9,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class = "header">
+      <a href="#home" class="Logo">
+        <img src= "./images/logo.png" alt = "Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a href="#" class="is-active nav__link" >
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="is-active nav__link" >
```
для nav__link задай висоту в 60пікселів, а для is-active потрібно створити псевдоелемент after (синя риска) і абсолютно його спозиціонувати (position: absolute)

Reply:
Віталій, дякую за дуже поширене пояснення)